apps/debugger/src/app/debugger.ts

```diff
@@ -0,0 +1,27 @@
+import { PluginClient } from "@remixproject/plugin";
+import { createClient } from "@remixproject/plugin-webview";
+import { DebuggerApiMixin, RawLocation, Sources, Asts, LineColumnLocation, 
+  onBreakpointClearedListener, onBreakpointAddedListener, onEditorContentChanged, TransactionReceipt, CompilerAbstract} from '@remix-ui/debugger-ui'
+
+export class DebuggerClientApi extends DebuggerApiMixin(PluginClient) {  
+  constructor () {
+    super()    
+    createClient(this as any)
+    this.initDebuggerApi()
+  }
+
+  offsetToLineColumnConverter: { offsetToLineColumn: (sourceLocation: RawLocation, file: number, contents: Sources, asts: Asts) => Promise<LineColumnLocation> }
```

Comment:
To avoid to be out of sync with the interface you can do :
```typescript
offsetToLineColumnConverter: IDebuggerApi['offsetToLineColumnConverter']
```